docker_custodian/docker_gc.py

```diff
@@ -151,10 +158,35 @@ def cleanup_images(client, max_image_age, dry_run, exclude_set):
         images = filter_images_in_use_by_id(images, image_ids_in_use)
     images = filter_excluded_images(images, exclude_set)
 
+    if recent_image_age is not None:
+        images = without_recently_used_images(client, images, recent_image_age)
+
     for image_summary in reversed(list(images)):
         remove_image(client, image_summary, max_image_age, dry_run)
 
 
+def without_recently_used_images(client, images, recent_image_age):
+    exclude = set()
+    for event in client.events(since=recent_image_age,
```

Comment:
Is since right here? Shouldn't this be a time not a timedelta?

Also a tiny nit: if this is a time can we make since+until the same type, seems they will accept either time or int but just for consistency.

Reply:
yeah, timedelta is wrong  name there, it's actually converted into datetime: https://github.com/Yelp/docker-custodian/blob/master/docker_custodian/args.py#L15

Reply:
Heh, of course!